Procfile

```diff
@@ -1,5 +1,5 @@
 thin:                      bundle exec rails s -p $PORT
-sidekiq:                   bundle exec sidekiq -q miq_bot
+sidekiq:                   bundle exec sidekiq -L log/sidekiq.log -q miq_bot
```

Comment:
Even though I agree this makes development easier, I believe @Fryguy or @brandondunne said this messes up the deployment mechanism we use on the actual bot.  @Fryguy @brandondunne Can we use an environment variable instead of a hardcoded path like we do the with thin port?